lib/matplotlib/axes/_base.py

```diff
@@ -2431,6 +2438,19 @@ def redraw_in_frame(self):
     def get_renderer_cache(self):
         return self._cachedRenderer
 
+    # layoutboxes: allow us to set these after instantiation.
+    def set_layoutbox(self, layoutbox):
+        """
+        Set the layoutbox for this axis...
+        """
+        self.layoutbox = layoutbox
+
+    def set_layoutboxpos(self, layoutbox):
+        """
+        Set the spinelayoutbox for this axis...
+        """
+        self.poslayoutbox = layoutbox
```

Comment:
layoutboxpos or poslayoutbox?  In any case I would suggest making the actual attribute private (above too) (or at the very least it should have the same name as given in the setter).